client/src/pages/messaging/index.js

```diff
@@ -79,7 +79,7 @@ const Chat = () => {
     myKeyRef.current = _keyPair;
   };
 
-  const handleSubmit = (e) => {
+  const handleSubmit = (e, imgUrl = '') => {
```

Comment:
It looks like design inconsistency. 
Either keep both text and imageUrl at the parent state or, pass both of them in the function argument.